src/Qwiq.Core.Soap/QueryDefinition.cs

```diff
@@ -0,0 +1,10 @@
+namespace Microsoft.Qwiq.Client.Soap
+{
+    public class QueryDefinition : Qwiq.QueryDefinition
+    {
+        public QueryDefinition(TeamFoundation.WorkItemTracking.Client.QueryDefinition queryDefinition)
```

Comment:
Should this not be `public`? The constructor exposes the underlying TFS type.

Reply:
Should be internal, will update.

Reply:
updated